### PR TITLE
Update OCICNI vendor and plugin directories

### PR DESCRIPTION
**Pull request by mheon**
Update the OCICNI vendor to pick up a new version which returns CNI results which returns information from CNI about the details of the network namespace that was configured. This should let us get more networking configuration information out of containers.

Also, add another location to look for CNI plugins by default. The upstream plugins default to installing into /opt/ so this should make using libpod with existing CNI installs easier.

### CODE REVIEW

**Comment by mheon** [comment]
Travis looks like a network-related flake

File: libpod/networking.go [hunk 1]
```diff
@@ -38,7 +38,8 @@ func (r *Runtime) createNetNS(ctr *Container) (err error) {
 
 	podNetwork := getPodNetwork(ctr.ID(), ctr.Name(), ctrNS.Path(), ctr.config.PortMappings)
 
-	if err := r.netPlugin.SetUpPod(podNetwork); err != nil {
+	_, err = r.netPlugin.SetUpPod(podNetwork)
+	if err != nil {
```
Comment (rhatdan) [inline libpod/networking.go]: nit: Still can be on single line.

**Comment by rhatdan** [comment]
One Nit.

File: libpod/networking.go (hunk 1 above)
Comment (mheon) [reply libpod/networking.go]: We're going to need the first return (CNI result with IP address and other info) eventually, so I figured I would put it on a new line

**Comment by rhatdan** [comment]
Ok LGTM
@baude @umohnani8 @TomSweeneyRedHat PTAL

**Comment by umohnani8** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by rhatdan** [comment]
Meging since bot seems to be offline.

**Pull request merged**